.travis.yml

```diff
@@ -12,11 +12,11 @@ compiler:
 notifications:
     email:
         recipients:
-            - su2code-dev@lists.stanford.edu
+            - aero_amit@hotmail.com         
   
 branches:
     only:
-        - develop
+        - AUSMPLUSUP
```

Comment:
This also.

Reply:
Reverted back this as well.

Thanks@EduardoMolina